src/components/Overlay/Overlay.scss

```diff
@@ -15,6 +15,7 @@
   right:  0;
   top:    0;
   z-index: $ms-zIndex-Overlay;
+  cursor: pointer;
```

Comment:
Is this suggesting using the hand pointer?

Sent from Outlook for iPhone<https://aka.ms/wp8k5y>

________________________________
From: David Zearing <notifications@github.com>
Sent: Thursday, September 1, 2016 4:36:42 PM
To: OfficeDev/office-ui-fabric-react
Subject: Re: [OfficeDev/office-ui-fabric-react] Mgodbolt/overlay pointer (#234)


In src/components/Overlay/Overlay.scss<https://github.com/OfficeDev/office-ui-fabric-react/pull/234#discussion_r77272927>:

> @@ -15,6 +15,7 @@
>    right:  0;
>    top:    0;
>    z-index: $ms-zIndex-Overlay;
> +  cursor: pointer;


Ummm I don't think we should do this. This will make all overlays look like a clickable target?

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/OfficeDev/office-ui-fabric-react/pull/234/files/c3645b9f93923c67d8c92f7d40a6a0eb9e90b4bc#r77272927>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATa3HY9e188cFKMu0H6hVPsegegm-MFuks5ql2GKgaJpZM4JzNFk>.